src/js/firstparties/twitter.js

```diff
@@ -80,6 +93,26 @@ function unwrapSpecialTwitterURLs() {
   });
 }
 
-unwrapSpecialTwitterURLs();
+function unwrapSpecialTwitterPwaURLs() {
+  try {
+    // iterate users object (usually should only be one, with some random ID?)
+    for (const entityKey of Object.keys(window.__INITIAL_STATE__.entities.users.entities)) {
```

Comment:
Getting an error here: "Cannot read property 'entities' of undefined".

Reply:
Ah yeah, that `wrappedJSONObject` had a reason. And anyway, this is why this is in a try box, if it fails, we don't care. That feature is not important.